crypto/crypto/src/traits.rs

```diff
@@ -252,6 +252,7 @@ pub trait Uniform {
         R: SeedableRng + RngCore + CryptoRng;
 
     /// Generate a random key using the shared TEST_SEED
+    // TODO: take an RNG
     fn generate_for_testing() -> Self
```

Comment:
No, I do not agree with this. We already have an API that takes in an RNG. Let's not have two APIs that do the same thing...

Reply:
ok, I'll rename to `generate_same_testing_keypair` then :D

Reply:
(not in this PR)